webapp/Connector/src/store/Explorer.js

```diff
@@ -7,5 +7,200 @@ Ext.define('Connector.store.Explorer', {
 
     extend: 'LABKEY.app.store.OlapExplorer',
 
-    subjectName: 'Subject'
+    subjectName: 'Subject',
+
+    //Override: Mostly the same as OlapExplorer, see code inside //DataSpace override for actual overrides
+    loadDimension : function(selections, altRequestDimNamedFilters) {
```

Comment:
This is a lot of copied code for an additional line. Could consider firing an event in the original or adding a stub method to allow preprocessing of a dimension query results. I realize this would mean a change to release17.1, however, could do in hosting17.1 and run CDS off of that.